orderbook/src/database/orders.rs

```diff
@@ -321,7 +333,11 @@ mod tests {
             signing_scheme: DbSigningScheme::Eip712,
         };
 
-        assert_eq!(order_row.calculate_status(), OrderStatus::Open);
+        assert_eq!(
+            order_row.calculate_status(),
+            OrderStatus::Open,
+            "Open - sell (filled - 0%)"
+        );
```

Comment:
The third argument to the assert! macros is the panic message that is printed when the assert fails. I don't think the message is useful here and would put this as a comment above if it is there to help readers understand what is being tested. Not a strong opinion, just feels slightly weird.

Reply:
I've added them because there are many asserts in this single test function.
When it fails, it doesn't tell me which one did, unless I'm missing something?

That's why I've added the panic message.

Reply:
Well, I did miss the line number on the error, my bad.
Will revert the change